README.md

```diff
@@ -120,6 +127,15 @@ From the base of the pyAMReX source directory, execute:
 python3 -m pip install -v .
 ```
 
+On successful installation, you can run the unit tests (assuming `pytest` is
+installed)
```

Comment:
For small diffs, I usually write one sentence per line in `.md` and `.rst` (like in LaTeX docs):
```suggestion
On successful installation, you can run the unit tests (assuming `pytest` is installed)
```